File: app/src/main/java/com/ikmr/banbara23/yaeyama_liner_checker/model/weather/Weather.java
package com.ikmr.banbara23.yaeyama_liner_checker.model.weather;

import java.util.List;

/**
 * 天気 一日分
 */
public class Weather {

    private String date;
    private List<Table> table;
    private Temperature temperature;
    private String wave;
    private String weather;
    private String wind;

    public String getDate() {
        return this.date;
    }

    public void setDate(String date) {
        this.date = date;
    }

    public List<Table> getTable() {
        return table;
    }

    public void setTable(List<Table> table) {
        this.table = table;
    }

    public Temperature getTemperature() {
        return this.temperature;
    }

    public void setTemperature(Temperature temperature) {
        this.temperature = temperature;
    }

    public String getWave() {
        return this.wave;
    }

    public void setWave(String wave) {
        this.wave = wave;
    }

    public String getWeather() {
        return this.weather;
    }

    public void setWeather(String weather) {
        this.weather = weather;
    }

    public String getWind() {
        return this.wind;
    }

    public void setWind(String wind) {
        this.wind = wind;
    }

    @Override
    public String toString() {
        return "Weather{" +
               "date='" + date + '\'' +
               ", table=" + table +
               ", temperature=" + temperature +
               ", wave='" + wave + '\'' +
               ", weather='" + weather + '\'' +
               ", wind='" + wind + '\'' +
               '}';
    }
}

File: app/src/main/java/com/ikmr/banbara23/yaeyama_liner_checker/model/weather/Weather.kt
package com.ikmr.banbara23.yaeyama_liner_checker.model.weather

/**
 * 天気 一日分
 */
data class Weather(
    // 日付
    var date: String = "",
    // 3時間毎の天気
    var table: List<Table> = listOf(),
    // 気温（最低＋最高）
    var temperature: Temperature = Temperature(),
    // 波の高さ
    var wave: String = "",
    // 天気
    var weather: String = "",
    // 風速
    var wind: String = "",
)
